uk_bin_collection/uk_bin_collection/councils/EastDevonDC.py

```
@@ -1,87 +1,122 @@
-                collections = week_value.find_next_siblings("span")
-                for item in collections:
-                    x = item.text
-                    bin_type = item.text.strip()
-                    if len(bin_type) > 1:
-                        dict_data = {
+    return addresses[0]
+
+
+def _parse_calendar_page(url):
+    """Parse the existing calendar page by UPRN (legacy path)."""
+    page = requests.get(url)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)